OtripleS.Web.Api/Brokers/Storage/StorageBroker.Teacher.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Microsoft.EntityFrameworkCore;
```

Comment:
Add copyrights notice.
Change the name of the file to say: `StorageBroker.Teachers.cs` - do the same with the interface as well.